src/cms-content/locations/state.json

```diff
@@ -0,0 +1 @@
e":"Iowa","locationType":"state","fips":"19","stateCode":""},{"locationName":"Kansas","locationType":"state","fips":"20","stateCode":""},{"locationName":"Kentucky","locationType":"state","fips":"21","stateCode":""},{"locationName":"Louisiana","locationType":"state","fips":"22","stateCode":""},{"locationName":"Maine","locationType":"state","fips":"23","stateCode":""},{"locationName":"Maryland","locationType":"state","fips":"24","stateCode":""},{"locationName":"Massachusetts","locationType":"state","fips":"25","stateCode":""},{"locationName":"Michigan","locationType":"state","fips":"26","stateCode":""},{"locationName":"Minnesota","locationType":"state","fips":"27","stateCode":""},{"locationName":"Mississippi","locationType":"state","fips":"28","stateCode":""},{"locationName":"Missouri","locationType":"state","fips":"29","stateCode":""},{"locationName":"Montana","locationType":"state","fips":"30","stateCode":""},{"locationName":"Nebraska","locationType":"state","fips":"31","stateCode":""},{"locationName":"Nevada","locationType":"state","fips":"32","stateCode":""},{"locationName":"New Hampshire","locationType":"state","fips":"33","stateCode":""},{"locationName":"New Jersey","locationType":"state","fips":"34","stateCode":""},{"locationName":"New Mexico","locationType":"state","fips":"35","stateCode":""},{"locationName":"New York","locationType":"state","fips":"36","stateCode":""},{"locationName":"North Carolina","locationType":"state","fips":"37","stateCode":""},{"locationName":"North Dakota","locationType":"state","fips":"38","stateCode":""},{"locationName":"Northern Mariana Islands","locationType":"state","fips":"69","stateCode":""},{"locationName":"Ohio","locationType":"state","fips":"39","stateCode":""},{"locationName":"Oklahoma","locationType":"state","fips":"40","stateCode":""},{"locationName":"Oregon","locationType":"state","fips":"41","stateCode":""},{"locationName":"Pennsylvania","locationType":"state","fips":"42","stateCode":""},{"locationName":"Puerto Rico","locationType":"state","fips":"72","stateCode":""},{"locationName":"Rhode Island","locationType":"state","fips":"44","stateCode":""},{"locationName":"South Carolina","locationType":"state","fips":"45","stateCode":""},{"locationName":"South Dakota","locationType":"state","fips":"46","stateCode":""},{"locationName":"Tennessee","locationType":"state","fips":"47","stateCode":""},{"locationName":"Texas","locationType":"state","fips":"48","stateCode":""},{"locationName":"Utah","locationType":"state","fips":"49","stateCode":""},{"locationName":"Vermont","locationType":"state","fips":"50","stateCode":""},{"locationName":"Virginia","locationType":"state","fips":"51","stateCode":""},{"locationName":"Washington","locationType":"state","fips":"53","stateCode":""},{"locationName":"West Virginia","locationType":"state","fips":"54","stateCode":""},{"locationName":"Wisconsin","locationType":"state","fips":"55","stateCode":""},{"locationName":"Wyoming","locationType":"state","fips":"56","stateCode":""}]}
```

Comment:
Same than before, since this file is not imported, is not going to be copied during the build process, we need to move it to `public`

Reply:
It's fine if this isn't imported - it's only used in the CMS